### PR TITLE
Change LSC referenced in Portal

### PR DESCRIPTION
**Pull request by billyhunt**
[Link to ZenHub issue.](link-goes-here)

## What does this change?

LSC requested changed to CRT Portal to try and limit the about of contact they were getting.  Here are those changes.

## Checklist:

- [x] Check all changes in the following document on the site.

[LSC.References.on.CRT.Reporting.Portal.RFlagg.edits.11.7.22.1.docx](https://github.com/usdoj-crt/crt-portal/files/10042557/LSC.References.on.CRT.Reporting.Portal.RFlagg.edits.11.7.22.1.docx)

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Review by JackRyan1989 — CHANGES_REQUESTED** [review]
Looks good except can you review the text in the EEOC referral letter? I think you're supposed to keep the text you removed. Not totally sure on that though.

File: crt_portal/cts_forms/response_templates/crt_eeoc_referral_letter.md [hunk 1]
```diff
@@ -118,17 +118,13 @@ American Bar Association
     www.americanbar.org/groups/legal_services/flh-home
     Contact by phone: 1-800-285-2221
 
-TO FIND...
-
-A personal attorney for low-income individuals
```
[the remark this answers is not in the document]
Comment (billyhunt) [reply crt_portal/cts_forms/response_templates/crt_eeoc_referral_letter.md:123]: Good catch!

**Review by JackRyan1989 — APPROVED** [review]
Looks good to me!!

**Review by ilodidoj — APPROVED** [review]
@billyhunt passed local docker testing.  Ready to merge for DEV.

**Pull request merged**